libs/sdk-ui-dashboard/src/model/tests/Dashboard.fixtures.ts

```diff
@@ -85,7 +85,17 @@ export const ComplexDashboardLayout = ComplexDashboardWithReferences.dashboard.l
 
 export const ComplexDashboardWidgets = {
     FirstSection: {
+        /**
+         * KPI with date filter disabled
+         */
         FirstKpi: ComplexDashboardWithReferences.dashboard.layout!.sections[0].items[0].widget! as IKpiWidget,
+        /**
+         * KPI with all filters disabled
+         */
+        ThirdKpi: ComplexDashboardWithReferences.dashboard.layout!.sections[0].items[2].widget! as IKpiWidget,
+        /**
+         * KPI with all filters enabled
```

Comment:
One of these comments might no be correct, one of the KPIs now has some filters ignored.

Reply:
yep. the comments should match the reality

- first kpi: no ignores & no date dataset set for kpi = date filter disabled, attribute filter ok
- third kpi: attr filter ignores & no date dataset set for KPI = all filters disabled
- last kpi: no attr filter ignores & date dataset set for kpi = all filters enabled

Reply:
I am stupid, I read all of these as enabled 🤦